CHANGELOG.md

```diff
@@ -73,6 +73,10 @@ Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-depl
 - Added `OnClientOwnershipGained` and `OnClientOwnershipLost` events on Actors and ActorComponents. These events trigger when an Actor is added to or removed from the ownership hierarchy of a client's PlayerController.
 - You can now generate valid schema for classes that start with a leading digit. The generated schema class will be prefixed with `ZZ` internally.
 - Handover properties will be automatically replicated when required for load balancing. `bEnableHandover` is off by default.
+- Worker types are now defined in the runtime settings.
```

Comment:
This would be a breaking change for users who need to move their worker types right?

Reply:
No, because we already had a hidden array which contained them. I just exposed it. Let me dive back into this, it's been a while.

Reply:
There is a settings modification though, the worker configuration Array in the editor settings is changed to a map, so there is a bit of code which does the conversio in USpatialGDKEditorSettings::SetRuntimeWorkerTypes